January/26/dave_gelperin/info.yaml

```diff
@@ -0,0 +1,13 @@
+title: David Gelperin
+description: |
+    On January 26, 1940 was born David Gelperin.
+    He is an software engineer and recieved PhD in Computer Science from Ohio State University.
```

Comment:
2nd sentence should be of the form "He is <nationality> <occupation>".

Reply:
Please explain why do you mean by this ?

Reply:
Argh, GitHub ate my formatting. It should read `"He is <nationality> <occupation>".`

Reply:
Not sure about this, I can only assume, could not find the info

Reply:
https://www.hodroffepsteinmemorialchapels.com/obituaries/David-Gelperin/#!/Obituary - born and lived most of his live in the USA. I think we can safely assume he was an American. 

occupation - software test engineer, software tester, quality consultant,  etc. That's generic enough given there isn't a specific profession listed.